README.md

```diff
@@ -28,31 +28,29 @@ protocol.
 - [**Useful Links**](#useful-links)
 
 
-## How To Use
+## How to use LSP4IntelliJ
```

Comment:
```suggestion
## How To Use
```

Reply:
Shall we keep this as it is? :) Since it is a common title used by another plugins and libraries as well

Reply:
+1